ansible/override.yml.example

```diff
@@ -0,0 +1,23 @@
+---
+# This is an example override file, which can be used as a template for your
+# own override file.
+
+node_types:
```

Comment:
Each of these could do with a comment describing what it is. Could just copy from host_vars.